src/test/kotlin/nl/info/zac/zaak/ZaakServiceTest.kt

```diff
@@ -348,7 +348,7 @@ class ZaakServiceTest : BehaviorSpec({
         )
         val identification = "fakeBSN"
         val roleAdviseur = createRolNatuurlijkPersoon(
-            zaaktypeURI = zaak.zaaktype,
+            zaakURI = zaak.zaaktype,
```

Comment:
Has this parameter name changed? So now the zaak type URI is referenced by a parameter named `zaakURI` that doesn't make sense and looks confusing to me 😞

Reply:
Ah this test is wrong. Good catch. It always was the zaak URI, not the zaaktype URI, but it was incorrectly named in the test fixtures. And wrongly used in some tests apparently.. :-(

Reply:
This test succeeds no matter what you fill in for the zaakURI. In that sense it's not a great test, but oh well.